assets/app/scripts/controllers/create.js

```diff
@@ -47,33 +119,198 @@ angular.module('openshiftConsole')
       return false;
     }
 
-    function updateTemplates() {
-      // Check if we've loaded templates from both the openshift and project namespaces.
-      $scope.templatesLoaded = openshiftTemplates && projectTemplates;
-      $scope.instantApps = {};
-      $scope.otherTemplates = {};
-
-      // Categorize templates as instant apps or "other."
-      var categorizeTemplates = function(template) {
-        var uid = uidFilter(template);
-        if (isInstantApp(template)) {
-          $scope.instantApps[uid] = template;
+    function matchesKeyword(name, tags, keyword) {
```

Comment:
I like the idea of searching description too (if even an option, which sounds complicated).

Perhaps we(I) could help create that "guidance" document around writing templates so they look good in Web UI. Could add this guidance when writing description to remove this false positive hits.